client/blocks/reader-related-card-v2/docs/example.jsx

```diff
@@ -56,24 +56,76 @@ const smallItems = [
+	{
+		post: {
+			ID: 3,
+			global_ID: 3,
+			site_ID: 3,
+			author: {
+				name: "ConservativeJoe",
+			},
+			date: "2016-09-27T08:20:33+00:00",
+			modified: "2016-09-27T08:20:37+00:00",
+			title: "Why Trump should Win the American Election.",
+			reading_time: 152,
+			better_excerpt: "This presidential race has, to be blunt, a cluster bomb of attacks, lies and outright nonsense rhetoric that has astounded many and made people curse at their screens, newspapers and media agencies they follow.",
+			short_excerpt: "This presidential race has, to be blunt, a cluster bomb of attacks, lies and outright nonsense rhetoric that has astounded many and made people curse at their…",
+			canonical_image: {
+					uri: "https://freedomsandtruth.files.wordpress.com/2016/09/trump-rally-in-vegas-getty-640x480.jpg?w=720&quality=80&strip=info",
+			}
+		},
+		site: {
+			title: 'All the catsss'
+		}
+	},
+	{
+		post: {
+			ID: 4,
+			global_ID: 4,
+			site_ID: 4,
+			author: {
+				name: "LiberalJoe",
+			},
+			date: "2016-09-27T08:20:33+00:00",
+			modified: "2016-09-27T08:20:37+00:00",
+			title: "Why Clinton should Win the American Election.",
+			reading_time: 152,
+			better_excerpt: "This presidential race, has, to be blunt, a lorem ipsum of devdocs demos.  HRC deserves the win because shes been playing the game longer than anybody, especially longer than trump. And she's tough.",
+			short_excerpt: "This presidential race has, to be blunt, a cluster bomb of attacks, lies and outright nonsense rhetoric that has astounded many and made people curse at their…",
+			canonical_image: {
+					uri: "https://lh4.googleusercontent.com/-eXKU4UhFusI/AAAAAAAAAAI/AAAAAAAAATA/1QahWqsqd-I/s0-c-k-no-ns/photo.jpg",
+			}
+		},
+		site: {
+			title: '99 Problems'
+		}
+	},
+];
+
 const RelatedPostCardv2Example = () => (
 	<div className="design-assets__group">
 		<Card>
-			<div className="reader-related-card-v2__container">
-				<h1 className="reader-related-card-v2__heading">More in <a href="#" className="reader-related-card-v2__link">Cats and Cats</a></h1>
-				<div className="reader-related-card-v2__posts is-site-post">
-					{ smallItems.map( item => <RelatedPostCard key={ item.post.global_ID } post={ item.post } site={ item.site } /> ) }
-				</div>
-			</div>
-		</Card>
-
-		<Card>
-			<div className="reader-related-card-v2__container">
-				<h1 className="reader-related-card-v2__heading">More in <a href="#" className="reader-related-card-v2__link">WordPress.com</a></h1>
-				<div className="reader-related-card-v2__posts is-global-post">
-					{ smallItems.map( item => <RelatedPostCard key={ item.post.global_ID } post={ item.post } site={ item.site } /> ) }
-				</div>
-			</div>
+		<div className="reader-related-card-v2__blocks is-same-site">
+			<h1 className="reader-related-card-v2__heading">More in <a className="reader-related-card-v2__link">Cats and Furballs</a></h1>
+				<ul className="reader-related-card-v2__list">
+					{ moreOnSameSite.map( item =>
```

Comment:
Should the arrow function body be wrapped with parens since it's multi-line? Like so:

```js
moreOnSameSite.map( item => (
    <Component />
)
```

There's a PR open to change up some of the JS guidelines so I'm not so sure how strict it'll be on wrapping blocks and return values so this is just a suggestion :)